Library/Homebrew/cmd/audit.rb

```diff
@@ -668,6 +668,22 @@ def audit_line(line, lineno)
     end
   end
 
+  def audit_prefix_has_contents
+    return unless formula.prefix.directory?
+
+    Pathname.glob("#{formula.prefix}/**/*") do |file|
+      next if file.directory?
+      basename = file.basename.to_s
+      return unless Metafiles.copy?(basename) || %w[.DS_Store INSTALL_RECEIPT.json].include?(basename)
```

Comment:
No it will change the logic. That's the reason @DomT4's test is failing. 

However if you want. I can change it into:
```ruby
next if ...
next if ...
return
```